src/components/Dice.jsx

```diff
@@ -0,0 +1,13 @@
+import PropTypes from "prop-types"
+
+export default function Dice({selected=false, children, freeze}) {
+    return <div onClick={freeze} className={`w-9 h-9 rounded-[4px] shadow-dice ${selected ? "bg-[#59E391]": "bg-white"} flex items-center justify-center`}>
+        {children}
+    </div>
```

Comment:
For good and structured code I think it would be better if you format this component with prettier.